include_list.txt

```diff
@@ -0,0 +1,15 @@
+# This is a list of Python additional packages needed. This file is used with pip.
+
+# ----- IDF v4.3 and v4.4 -----
+# cffi 1.15 supports Python 3.6+ only
+cffi<1.15;python_version<'3.6'
+
+# esptool requirements (see components/esptool_py/esptool/setup.py)
+reedsolo>=1.5.3,<=1.5.4
```

Comment:
If we add this requirement in addition to the latest esptool's `"reedsolo>=1.5.3,<1.8"` (https://github.com/espressif/esptool/blob/master/setup.py#L131C9-L131C31) then won't that mean that we will build only `reedsolo==1.5.4`?

If an ESP-IDF v5.x user will install esptool then it will pull in `reedsolo==1.7.0` and the wheel will be built. Or am I wrong?

IMO, you can remove this restriction as well. `reedsolo` is a pure-Python package for which we don't need to provide a wheel.